client/driver/rkt.go

```diff
@@ -0,0 +1,206 @@
+        // Only enable if we are root when running on non-windows systems.
+        if runtime.GOOS != "windows" && syscall.Geteuid() != 0 {
+                d.logger.Printf("[DEBUG] driver.rkt: must run as root user, disabling")
+                return false, nil
+        }
+
+        outBytes, err := exec.Command("rkt", "version").Output()
+        if err != nil {
+                return false, nil
+        }
+        out := strings.TrimSpace(string(outBytes))
+
+        rktMatches := reRktVersion.FindStringSubmatch(out)
+        appcMatches := reRktVersion.FindStringSubmatch(out)
+        if len(rktMatches) != 2 || len(appcMatches) != 2 {
+                return false, fmt.Errorf("Unable to parse Rkt version string: %#v", rktMatches)
+        }
+
+        node.Attributes["driver.rkt"] = "1"
+        node.Attributes["driver.rkt.version"] = rktMatches[0]
+        node.Attributes["driver.appc.version"] = appcMatches[1]
+
+        return true, nil
+}
+
+// Run an existing Rkt image.
+func (d *RktDriver) Start(ctx *ExecContext, task *structs.Task) (DriverHandle, error) {
+        trust_prefix, ok := task.Config["trust_prefix"]
+        if !ok || trust_prefix == "" {
+                return nil, fmt.Errorf("Missing trust prefix for rkt")
+        }
+        trust_args := []string{
+                "sudo",
+                "rkt",
+                "trust",
+                "--prefix=" + trust_prefix,
+        }
+        // Add the given trust prefix
+        var outBuf, errBuf bytes.Buffer
+        cmd := exec.Command(trust_args[0], trust_args[1:]...)
+        cmd.Stdout = &outBuf
+        cmd.Stderr = &errBuf
+        d.logger.Printf("[DEBUG] Starting rkt command: %q", strings.Join(trust_args, " "))
+        if err := cmd.Start(); err != nil {
+                return nil, fmt.Errorf(
+                        "Error running rkt: %s\n\nOutput: %s\n\nError: %s",
+                        err, outBuf.String(), errBuf.String())
+        }
+        d.logger.Printf("[DEBUG] Added trust prefix: %q", trust_prefix)
+
+        name, ok := task.Config["name"]
+        if !ok || name == "" {
+                return nil, fmt.Errorf("Missing ACI name for rkt")
+        }
+        run_args := []string{
+                "sudo",
+                "rkt",
+                "run", "--interactive", name,
+        }
+        // Run the ACI
+        var aoutBuf, aerrBuf bytes.Buffer
+        acmd := exec.Command(run_args[0], run_args[1:]...)
+        acmd.Stdout = &aoutBuf
+        acmd.Stderr = &aerrBuf
+        d.logger.Printf("[DEBUG] Starting rkt command: %q", strings.Join(run_args, " "))
+        if err := acmd.Start(); err != nil {
+                return nil, fmt.Errorf(
+                        "Error running rkt: %s\n\nOutput: %s\n\nError: %s",
+                        err, aoutBuf.String(), aerrBuf.String())
+        }
+        d.logger.Printf("[DEBUG] Started ACI: %q", name)
+        h := &rktHandle{
+                proc:   cmd.Process,
```

Comment:
Should this be `acmd`? It's using the `cmd` from line 92  above, where we invoke `rkt trust`